app/assets/stylesheets/custom.css.scss

```diff
@@ -71,3 +71,28 @@ h4 {
   text-align: center;
   margin-bottom: 20px;
 }
+
+.item-create {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
+  border: 1em;
+  border-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.